packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -70,6 +70,14 @@ export interface Ec2ServiceProps extends BaseServiceOptions {
    * @default false
    */
   readonly daemon?: boolean;
+
+  /**
+   * Specifies whether to propagate the tags from the task definition or the service to the tasks in the service.
+   * Tags can only be propagated to the tasks within the service during service creation.
+   *
+   * @default SERVICE
```

Comment:
Can you actually add the entire enum value here? `PropagatedTagSource.SERVICE`

Reply:
Done